src/responses.js

```diff
@@ -0,0 +1,24 @@
+function respondWithError(res, errors) {
+  res.statusCode = 400;
+  res.statusMessage = 'Bad request';
```

Comment:
```suggestion
function respondWithBadRequestError(res, errors) {
  res.statusCode = 400;
  res.statusMessage = 'Bad request';
```

Reply:
errors can be different so for better reusability better to either create helper for a particular error or pass the error code as parameter